fetcher/fetcher.go

```diff
@@ -1,298 +1,14 @@
 package fetcher
 
 import (
-	"context"
 	"fmt"
 	"io/ioutil"
 	"os"
 	"path/filepath"
 	"strings"
-	"sync"
 	"time"
-
-	"github.com/algorand/go-algorand-sdk/v2/client/v2/algod"
-	"github.com/algorand/go-algorand/protocol"
-	"github.com/algorand/go-algorand/rpcs"
-	"github.com/algorand/indexer/util/metrics"
-	log "github.com/sirupsen/logrus"
 )
 
-// Fetcher is used to query algod for new blocks.
-type Fetcher interface {
```

Comment:
@barnjamin if you still use this function, now would be a good time to migrate it to a library.